.github/CODEOWNERS

```diff
@@ -1438,7 +1440,7 @@ x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor te
 # AppEx Platform Services Security
 //x-pack/test_serverless/api_integration/test_suites/common/security_response_headers.ts  @elastic/kibana-security
 /x-pack/test/api_integration/apis/es  @elastic/kibana-security
-
+/x-pack/test/functional_cors @elastic/kibana-security
```

Comment:
It looks like the Core team created these tests. @elastic/kibana-core would you prefer to retain ownership or transfer ownership to us? We're fine taking ownership if that is your preference.

@wayneseymour there is an existing directive for `x-pack/test/functional_cors/plugins/kibana_cors_test` that is assigned to us, which should be removed, as the directive you're adding is broader scope. FWIW we're not sure how we got ownership of this one.

This brings up the point that directives will override each other and there is no feasible way to test for this. So when adding new directives, it's important to check for any overlapping directives to remove redundancies or unwanted overrides.

Reply:
Great point kind sir, great point.  I'll do my best to double check!

Reply:
> we're not sure how we got ownership of this one.

Looks like it was assigned here: https://github.com/elastic/kibana/pull/148130
There are too many files in that pr to hunt down with proper provocation.
That said, if that all checks out, this means that [x-pack/test/functional_cors/plugins/kibana_cors_test @elastic/kibana-security](https://github.com/elastic/kibana/blob/99b9b5ef59b3f23e95b23d041f94bd2a67fd744c/.github/CODEOWNERS#L555) is a default assignment.  IIRC that means this is a default assignment (assigned via a kibana.jsonc with kbn-generate), that can be overriden by re-assigning [lower](https://github.com/elastic/kibana/blob/99b9b5ef59b3f23e95b23d041f94bd2a67fd744c/.github/CODEOWNERS#L1019) in the codeowners file.

Reply:
Dropping the disputed line for now. Will follow up in another pr later

Reply:
Thanks Rudolf!